src/jrd/Database.cpp

```diff
@@ -47,6 +47,14 @@
 
 using namespace Firebird;
 
+namespace
+{
+	void unableToRunSweepException(const Arg::Gds& reason)
```

Comment:
Passing ISC_STATUS to exception raise will make code simpler, cleaner and cheaper.

Reply:
Done